extensions/functions_arithmetic.yaml

```
@@ -1071,6 +1071,54 @@ aggregate_functions:
             value: fp64
         nullability: DECLARED_OUTPUT
         return: fp64?
+  - name: "corr"
+    description: >
+      Calculates correlation of two set of values. 
+      If there is no input, null is returned.
+    impls:
+      - args:
+          - name: rounding
+            options: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
+            required: false
+          - value: fp32
+          - value: fp32
+        nullability: DECLARED_OUTPUT
+        return: fp32?
+      - args:
+          - name: rounding
+            options: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
+            required: false
+          - value: fp64
+          - value: fp64
+        nullability: DECLARED_OUTPUT
+        return: fp64?
+  - name: "mode"
+    description: Calculates mode for a set of values.
```

Please update for mode as well;

```suggestion
    description: >
      Calculates mode for a set of values.
      If there is no input, null is returned.
```